configs/beaglebone_black_base_config

```diff
@@ -1,39 +1,70 @@
-# Binaries generated with the following script:
-#    https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/build-uboot-bbb.sh
-
-# There are reported issues with GRUB bootloader integration, fallback to U-boot.
-MENDER_GRUB_EFI_INTEGRATION=n
+# At the time of writing, the binaries below were uploaded manually. However,
+# they are simply a vanilla U-Boot build of v2020.07, from Ubuntu 18.04, using
+# these commands:
```

Comment:
Should we update https://github.com/mendersoftware/uboot-mender along with this?

Reply:
What do you mean? To upload the binaries? I don't think it's worth the effort. This is a one-shot upload which only applies to this specific image which is available today. Everything we need is already upstream, it's just not in the image yet.

Reply:
Do we need the ability to rebuild this reliably?

Reply:
No, this is a one time thing, we want to use the U-Boot image which is already in the image. The only reason it is done for this specific image is because their image is old and out of date.